<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamController</class>
 <widget class="QWidget" name="ExamController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit_word1"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_translation1">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_word2"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_translation2">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit_word3"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_translation3">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_word4"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_translation4">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit_word5"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_translation5">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Fill in the correct translation </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_backToMenu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_checkResults">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Check answers</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="horizontalSpacing">
     <number>12</number>
    </property>
    <property name="verticalSpacing">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="1">
     <widget class="QLabel" name="label_correctAnswer1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_correctAnswer2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_correctAnswer3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_correctAnswer4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_correctAnswer5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
